Integrations/integration-VirusTotal-Private_API.yml

```diff
@@ -100,7 +100,12 @@ script:
             )
             res.raise_for_status()
 
-            return res.json()
+            if res.status_code == 200:
+                return res.json()
+            # 204 HTTP status code is returned when api rate limit has been exceeded
+            elif res.status_code == 204:
+                return_error("You've reached your API call quota. Contact your VirusTotal representative.")
```

Comment:
Just `You've reached your API call quota.` is enough

Reply:
talked with alex, it's what he recommended.